admin_solutions/index.adoc

```diff
@@ -17,5 +17,6 @@ In this initial version of the guide, you can learn how to:
 - xref:../admin_solutions/user_role_mgmt.adoc#share-templates-cluster[Share templates across the cluster]
 - xref:../admin_solutions/user_role_mgmt.adoc#create-cluster-admin[Create a cluster administrator account]
 - xref:../admin_solutions/authentication.adoc#admin-solutions-authentication[Configure authentication providers]
+- xref:../admin_solutions/certificate_management.adoc[Certificate Management]
```

Comment:
@lpsantil Now that I think of it, there are a few minor things that I need to do with this PR, but I can take care of those things for you by fetching your branch. I'll update the link as part of my process. Thanks!